### ISSUE TITLE
Possible mistake in tutorial docs

### ISSUE THREAD

**Issue by iandewancker** [blei-lab/edward#643]
Hey there, super cool project

I was reading the supervised classification example with GPs : http://edwardlib.org/tutorials/supervised-classification and I wondered if the line that defines the likelihood might have a slight mistake

p(yn ∣z, x​n) = Bernoulli ( yn ∣ logit−1(xn⊤z)). should actually be

p(yn ∣z, x​n) = Bernoulli ( yn ∣ logit−1(zn)) 

if z represents the latent functions values then we simply squash them through the sigmoid function (as in the Rasmussen text) and don´t need to take the dot product with xn.  
​​

**Comment by dustinvtran** [comment]
It needs to take the inputs to learn the input-output relationship. What you're describing is known as a GPLVM, which is a different model.

**Comment by iandewancker** [comment]
Ah, so perhaps the code snippet is describing the GPLVM?  My understanding is f will serve as analogous to z.
```
X = tf.placeholder(tf.float32, [N, D])
f = MultivariateNormalTriL(loc=tf.zeros(N), scale_tril=tf.cholesky(rbf(X)))
y = Bernoulli(logits=f)
```

**Comment by dustinvtran** [comment]
Oh, apologies I misunderstood you. You are correct. That should be changed to just be a "z", since that notation for "z" means the multivariate normal. Contribution to fix this is welcome.

**Comment by dustinvtran** [comment]
Fixed in #645
